From 10f8874df0de3831428dfacb7e6f084a1946ca6d Mon Sep 17 00:00:00 2001
From: Marty Jones <mj8263788@gmail.com>
Date: Wed, 6 Jan 2021 17:58:30 -0500
Subject: [PATCH] rockchip: enabled USB3 port on NanoPi NEO3

This enable the USB3 port on the FriendlyARM NanoPi NEO3

Signed-off-by: Marty Jones <mj8263788@gmail.com>
---
 arch/arm64/boot/dts/rockchip/rk3328-nanopi-neo3.dts | 9 +++++++++
 1 file changed, 9 insertions(+)

--- a/arch/arm64/boot/dts/rockchip/rk3328-nanopi-neo3.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3328-nanopi-neo3.dts
@@ -348,3 +348,12 @@
 &usb_host0_ohci {
 	status = "okay";
 };
+
+&usbdrd3 {
+	status = "okay";
+};
+
+&usbdrd_dwc3 {
+	dr_mode = "host";
+	status = "okay";
+};
